.pylintrc

```diff
@@ -75,7 +75,6 @@ enable=
 # --disable=W"
 disable=
     cyclic-import,  # re-enable once this no longer raises false positives
-    no-member,  # re-enable once this no longer raises false positives. This will become redundant after the min required version is 3.11
```

Comment:
@EugeneTorap personally I think it is preferable to keep this disables inline. We use the `useless-suppression` check so when we upgrade to Python 3.11 as a minimum the check will fire. It seems like an cleaner/safer cleanup mechanism that leaving notes in the code.

Reply:
@john-bodley Please, check this #24803 PR. We added this line there.

![image](https://github.com/apache/superset/assets/29536522/5b9f047d-0d73-4f12-87a4-26642dd5d350)

Reply:
Apologies @EugeneTorap. I misread this as an addition as opposed to a removal.